pycbc/strain.py

```diff
@@ -328,6 +328,20 @@ def from_cli(opt, dyn_range_fac=1, precision='single'):
             logging.info("Converting to float32")
             strain = (dyn_range_fac * strain).astype(pycbc.types.float32)
 
+    if opt.fake_strain_from_file:
+        logging.info("Generating Fake Strain from ASD file")
```

Comment:
Can you avoid the code duplication of the next few lines? Perhaps always calculate these quanitties before the if statements.